lib/bolt/cli.rb

```diff
@@ -51,7 +51,7 @@ def inventory
     end
     private :inventory
 
-    def help?(parser, remaining)
+    def help?(_parser, remaining)
```

Comment:
You can remove the `_parser` option since it's not used in this function anymore